README.md

```diff
@@ -8,21 +8,30 @@ docker build --tag h379wang/kim-model-conversion .
 docker run -it h379wang/kim-model-conversion:latest /bin/bash
 ```
 ----------
+### Get Datasets
+```
+cd kim_cnn
+./get_data.sh
```

Comment:
I tried running this, but the docker container doesn't have the python package "gensim" installed so the convert_bin2txt.py script can run - this package should be added to the docker container?

Reply:
Yes thanks for the reminder of adding the `pip install gensim==3.5.0` line to the docker makefile.
Note that the convert_bin2txt.py process of converting the whole GoogleNews-vectors-negative300.bin embedding can take a long time. For testing purpose if you only need a small subset of the embedding, then please update get_data.sh to the following code before docker build.

```
#!/bin/bash
mkdir data
cd data
wget http://ocp59jkku.bkt.clouddn.com/sst-1.zip
unzip sst-1.zip
mv word2vec.sst-1 word2vec.txt
wget https://raw.githubusercontent.com/Impavidity/kim_cnn/master/data/stsa.fine.dev.tsv
wget https://raw.githubusercontent.com/Impavidity/kim_cnn/master/data/stsa.fine.phrases.train.tsv
wget https://raw.githubusercontent.com/Impavidity/kim_cnn/master/data/stsa.fine.test.tsv
wget https://raw.githubusercontent.com/Impavidity/kim_cnn/master/vector_preprocess.py
```

Reply:
This script should also be made executable (`chmod +x get_data.sh`).